db/migrate/20200505123318_create_events.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :events do |t|
+      t.belongs_to :user
+      t.string :title
+      t.string :description
+      t.date :event_date
```

Comment:
The events table has creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

You should use t.references for the requested associations above.